array_methods_lab.js

```diff
@@ -0,0 +1,365 @@
+// # Exercises
+//
+// 1. Given an array of numbers, write a function that returns a new array where every
+// //element is incremented by 10.
+//
+// ```
+// Input: [1,2,3,4,5]
+// Output: [11,12,13,14,15]
+
+ console.log('=====answer#1======')
+
+let arr = [1,2,3,4,5]
+let incrementedArr = arr.map((currElem, index) => {
+
+  return( currElem += 10 )
+})
+ console.log(incrementedArr)
+
+// ```
+//
+// 2. Given an array, return a new array where every non-string element is converted
+// to an empty string.
+//
+// ```
+// Input: ["a", 123, "b", "c", {name: "cat"}]
+// Output: ["a", "", "b", "c", ""]
+
+console.log('========answer#2========')
+
+let arr1 = ["a", 123, "b", "c", {name: "cat"}]
+let allStringArr = arr1.map((crrElem, index) => {
+if (typeof crrElem === 'number' || typeof crrElem === 'object' ) {
+  return " "
+} else {
+  return  crrElem
+}
+
+})
+console.log(allStringArr)
+
```

Comment:
Great idea. However what if the typeof is a boolean? How can we make sure we make sure we account for any type that not a string?